Project.toml

```
@@ -1,6 +1,6 @@
 name = "StaticArraysCore"
 uuid = "1e83bf80-4336-4d27-bf5d-d5a4f845583c"
-version = "1.1.0"
+version = "1.1.1"
```

```suggestion
version = "1.2.0"
```
I prefer to bump minor version when moving new types here.